### DISCUSSION TITLE
How to check descendant class existence in Cypress?

### DISCUSSION THREAD

**Discussion by quad5** [cypress-io/cypress#16902] [Questions and Help] [answered]
Hi,

Is it possible to check descendant class using `should` in Cypress? At the moment, it seems like it only check class on the same level .

`cy.get('.meal-planning-title').should('not.have.class', '.notification-badge');`

<img width="855" alt="Untitled20" src="https://user-images.githubusercontent.com/2043287/121597795-f9336e80-c9f5-11eb-93f6-b99bfc189e9b.png">
<img width="663" alt="Untitled21" src="https://user-images.githubusercontent.com/2043287/121598241-78c13d80-c9f6-11eb-8ec6-9559502c7c9c.png">

**Comment by bahmutov** [comment, marked as answer]
You could use should with a callback function. You would get the jquery element and could use any logic inside the callback to verify the descendants
